src/System.Management.Automation/engine/remoting/server/OutOfProcServerMediator.cs

```diff
@@ -170,7 +168,10 @@ protected void OnCommandCreationPacketReceived(Guid psGuid)
 
                 _inProgressCommandsCount++;
 
-                tracer.WriteMessage("OutOfProcessMediator.OnCommandCreationPacketReceived, in progress command count : " + _inProgressCommandsCount + " psGuid : " + psGuid.ToString());
+                using (PowerShellTraceSource tracer = PowerShellTraceSourceFactory.GetTraceSource())
```

Comment:
Note that the instance `tracer` variable was replaced with this locally scoped variable to match the pattern used in the `OnClosePacketReceived` method. A disposable `tracer` instance variable would be more efficient, but the enclosing class currently is not `IDisposable`, so that would be a breaking change. Additionally, it's derived classes are being used in singleton fashion, and having disposable singletons is probably a discussion topic out of scope for this PR.